vpc-native/main.tf

```diff
@@ -30,10 +30,9 @@ resource "google_container_cluster" "cluster" {
     master_ipv4_cidr_block  = var.master_ipv4_cidr_block
   }
 
-  # Disable automatic kubernetes dashboard
   addons_config {
-    kubernetes_dashboard {
-      disabled = true
+    network_policy_config {
+      disabled = false
```

Comment:
Any reason not to make this the default in the module?

Reply:
I don't think so. This should be safe to have here, as we also have this set:
```  network_policy {
    enabled = true
  }
```
Oddly these both have to be set for network policies to be enabled. So this is a good thing to include in the default.